local/tractus-x-edc/config/supplier/data-plane.properties

```diff
@@ -21,6 +21,13 @@ edc.hostname=supplier-data-plane
 # HashiCorp vault related configuration
 edc.vault.hashicorp.url=http://vault:8200
 # set via docker-compose .env
-#edc.vault.hashicorp.token=
-#disable annoying logging - do we need this enabled?
 edc.vault.hashicorp.health.check.enabled=false
+edc.vault.hashicorp.token= ${VAULT_DEV_ROOT_TOKEN_ID}
+
+
+edc.datasource.edr.name=edr
```

Comment:
double blank + why enabled for dataplane but not controlplane?

Reply:
I just did some more testing and found out, that we actually dont need any of the edc.datasource.edr.* properties in our data plane config, since we're not using those features because we are having our own edrEndpoint for receiving and storing edr tokens.

Reply:
Maybe makes sense to keep them as outcommented for the beginning. I'm pretty sure there will be the day we need to switch to postgres and then need it